cloudferrylib/os/actions/check_cloud.py

```diff
@@ -12,13 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import time
+from cloudferrylib.base import exception
 from cloudferrylib.base.action import action
 from cloudferrylib.utils import utils as utl
 from keystoneclient.openstack.common.apiclient import exceptions as ks_exc
 from novaclient import exceptions as nova_exc
-from cloudferrylib.base import exception
 
 LOG = utl.get_log(__name__)
+LIMIT_RETRY_CHECK_INSTANCE = 25
```

Comment:
Where it is specified?
Here is the description of this option from `configs/config.ini`:
>  Number x API retries.
> Note: High number may considerably slow down migration process, but ensures
> retry.
https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/configs/config.ini#L103-L105

And also from `cfglib.py`:
> Number retry if except Performing error
https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/cfglib.py#L137

There is nothnig related to proxy-client there.